Source/appfat.cpp

```diff
@@ -11,43 +11,43 @@ namespace devilution {
 
 namespace {
 
-/** Set to true when a fatal error is encountered and the application should shut down. */
-bool terminating = false;
-/** Thread id of the last callee to FreeDlg(). */
-SDL_threadID cleanup_thread_id;
+	/** Set to true when a fatal error is encountered and the application should shut down. */

```

Comment:
These formatting changes come from nested `.clang-format` files that @AJenbo hasn't had a chance to delete yet.

Please revert all the formatting changes in the PR (in general formatting changes should not be mixed with code changes -- for ease of review and better commit history).

Reply:
I think it would be great to solve that issue. It would be good if making a code change doesn't break the formatting because the file was not correctly formatted before.

Reply:
Very close to doing so, I think there is only one file left to deal with.